code/HomepageForDomainExtension.php

```diff
@@ -1,80 +1,95 @@
 <?php
-class HomepageForDomainExtension extends DataExtension
-{
+namespace {
 
```

Comment:
@phalkunz any special reason not to add namespace. why global namespace?

Reply:
There's a reference to in `RootURLController`.

Reply:
I don't think we can be polluting the root namespace - the core module will need updating.